pygmt/src/sph2grd.py

```diff
@@ -0,0 +1,78 @@
+"""
+sph2grd - Compute grid from spherical harmonic coefficients
+"""
+from pygmt.clib import Session
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+from pygmt.io import load_dataarray
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    I="spacing",
+    R="region",
+    V="verbose",
+    b="binary",
+    h="header",
+    i="incols",
+    r="registration",
+    x="cores",
+)
+@kwargs_to_strings(I="sequence", R="sequence", i="sequence_comma")
+def sph2grd(data, **kwargs):
+    r"""
+    Create spherical grid files in tension of data.
+
+    Reads a spherical harmonics coefficient table with records of L, M,
+    C[L,M], S[L,M] and evaluates the spherical harmonic model on the
+    specified grid.
+
+    Full option list at :gmt-docs:`sph2grd.html`
+
+    {aliases}
+
+    Parameters
+    ----------
+    data : str or {table-like}
+        Pass in (x, y, z) or (longitude, latitude, elevation) values by
+        providing a file name to an ASCII data table, a 2D
+        {table-classes}.
```

Comment:
I have never used this module, but I believe it doesn't need (x, y, z) input.

Reply:
updated in 4f22a4e